build-system/tasks/presubmit-checks.js

```diff
@@ -414,6 +414,13 @@ var forbiddenTermsSrcInclusive = {
   '\\.webkitConvertPointFromPageToNode(?!_)': bannedTermsHelpString,
   '\\.changeHeight(?!_)': bannedTermsHelpString,
   '\\.changeSize(?!_)': bannedTermsHelpString,
+  'insertAmpExtensionScript(?!\\s*\/\\*OK\\*\\/)': {
```

Comment:
I think here per-file whitelisting is more appropriate. It is also safer as soon as we land support for per-directory OWNERS